### PR TITLE
Added initial README file

### PR DESCRIPTION
**Pull request by matthiasbeyer**
Add README.md

### CODE REVIEW

**Comment by neithernut** [comment]
ACK.

**Comment by matthiasbeyer** [comment]
Please review again and ACK for squash.

**Comment by neithernut** [comment]
ACK.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Waysome
+
+Note: Waysome is work in progress and not ready or useable yet.
+
+Waysome is a window manager using wayland. It is especially made for powerusers
+and people who like to configure their window manager very atomicly.
+
+Waysome is developed as semester project at
```
Comment (TheNeikos) [inline README.md]: it should be `as a semester`

**Comment by matthiasbeyer** [comment]
@TheNeikos ack for squashing?

**Comment by TheNeikos** [comment]
Yes

**Comment by matthiasbeyer** [comment]
Done.

**Comment by neithernut** [comment]
Re-ACK.

**Pull request merged**